### PR TITLE
Update helm chart to support actions metrics graceful termiantion

### PR DESCRIPTION
**Pull request by Smirl**
# Summary

- add lifecycle, terminationGracePeriodSeconds, and loadBalancerSource ranges to metrics server
- these were missed when copying from the other webhook server
- original PR adding them to the other webhook server is here https://github.com/actions/actions-runner-controller/pull/2305

### CODE REVIEW

**Comment by Smirl** [comment]
FYI @mumoshu . Thanks for the review.

File: charts/actions-runner-controller/Chart.yaml [hunk 1]
```diff
@@ -15,7 +15,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.23.1
+version: 0.23.2
```
Comment (mumoshu) [inline charts/actions-runner-controller/Chart.yaml]:
```suggestion
version: 0.23.1
```

I'd appreciate it if you could leave this as-is and let me coordinate the timing of the new chart release myself! (A change in this chart version results a new chart release on merge)

**Review by mumoshu — COMMENTED** [review]
Hey @Smirl! Thank you so much for your contribution. I managed to review this today and had only one comment. Would you mind checking it? Once it's addressed, this is good to be merged!

**Comment by Smirl** [comment]
Thanks @mumoshu. Committed your suggestion and rebased.

File: charts/actions-runner-controller/Chart.yaml [hunk 2]
```diff
@@ -15,7 +15,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.23.2
+version: 0.23.1
```
Comment (mumoshu) [inline charts/actions-runner-controller/Chart.yaml]:
```suggestion
version: 0.23.2
```

**Review by mumoshu — APPROVED** [review]
LGTM. Thank you very much for your contribution @Smirl!

**Pull request merged**